server/service/src/invoice/inbound_shipment/insert/generate.rs

```diff
@@ -49,6 +49,7 @@ pub fn generate(
         verified_datetime: None,
         linked_invoice_id: None,
         requisition_id: None,
+        mode: InvoiceRowMode::Store,
```

Comment:
Any invoices (which are not dispensary) have the store mode even when created in dispensary mode